backend/app/services/knowledge/code_wiki/scheduled_update.py

```
@@ -0,0 +1,390 @@
+            "MODEL_UNAVAILABLE: Code Wiki team is unavailable for this runner"
+        )
+    return runner
+
+
+def configure_scheduled_update(
+    db: Session, *, knowledge_base: Kind, data: CodeWikiAutomaticUpdateRequest
+) -> Kind:
+    """Create or update one scheduler row while serializing on its Code Wiki."""
+    knowledge_base = (
+        db.query(Kind)
+        .filter(Kind.id == knowledge_base.id, Kind.is_active == True)
```

Ruff (0.16.3)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)